generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.ts.ejs

```diff
@@ -16,14 +16,14 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { <%_ if (pagination !== 'no') { _%>HttpHeaders, <% } %>HttpResponse } from '@angular/common/http';
 <%_ if (pagination === 'pagination') { _%>
 import { ActivatedRoute, Router } from '@angular/router';
+import { combineLatest } from 'rxjs';
```

Comment:
this import seems unused

Reply:
Looks right. Wondering why eslint didn’t complained.

Reply:
This is used in `pagination-template`, so this import is needed and is under right condition.  
This is migrated from deleted row below.

Reply:
indeed ;)